flagengine/segments/models.go

```diff
@@ -33,7 +33,6 @@ func (m *SegmentConditionModel) MatchesTraitValue(traitValue string) bool {
 func (m *SegmentConditionModel) regex(traitValue string) bool {
 	match, err := regexp.Match(m.Value, []byte(traitValue))
 	if err != nil {
-		// TODO: better logging
 		log.Printf("WARNING: Invalid regex expression %v", m.Value)
```

Comment:
Shouldn't this line also be changed to use the provided logger instead of directly to stdout?

Reply:
Yes, good catch. @gagantrivedi are you able to update this in another PR?

Reply:
This will be removed (updating this to use provided logger will need a decent amount of code change with very little upside, at least right now)

Reply:
Nice!
We will probably never run in to this, but just wanted to make sure you were aware :)